ui/packages/catalog-ui-search/src/main/webapp/react-component/container/marionette-region-container/marionette-region-container.tsx

```diff
@@ -28,78 +28,75 @@ const RegionContainer = styled.div`
   width: 100%;
   height: 100%;
 `
-export default hot(module)(
-  class MarionetteRegionContainer extends React.Component<Props, {}> {
-    constructor(props: Props) {
-      super(props)
-    }
-    checkForElement: number
-    region: any
-    regionRef = React.createRef()
-    showComponentInRegion() {
-      if (this.props.view._isMarionetteView) {
-        this.region.show(this.props.view, {
-          replaceElement: this.props.replaceElement,
-        })
-      } else {
-        this.region.show(new this.props.view(this.props.viewOptions), {
-          replaceElement: this.props.replaceElement,
-        })
-      }
-    }
-    onceInDOM(callback: () => void) {
-      clearInterval(this.checkForElement)
-      this.checkForElement = window.setInterval(() => {
-        if (document.body.contains(this.regionRef.current as Node)) {
-          clearInterval(this.checkForElement)
-          callback()
-        }
-      }, intervalToCheck)
-    }
-    handleViewChange() {
-      this.resetRegion()
-      this.onceInDOM(() => {
-        this.showComponentInRegion()
+export class MarionetteRegionContainer extends React.Component<Props, {}> {
```

Comment:
React hot loader makes it so you can't extend ES6 classes...whatever it does when wrapping the component, hence exporting the class here. Default export is still there just moved to the bottom. Functionally there are no changes to `marionette-region-container` as compared to master only formatting is different due to `hot(module)(` business being moved.

Reply:
With the updates to use a modified FilterBuilderView, I think we can safely remove the changes here.